src/ddscxx/tests/SharedMemory.cpp

```diff
@@ -56,15 +64,51 @@ void make_sample_(HelloWorldData::Msg & sample, const int32_t cnt)
   sample.message(std::to_string(cnt));
 }
 
+char get_random_char()
+{
+  std::random_device rd;
+  std::mt19937 gen(rd());
```

Comment:
This could/should probably be `static` to create it only once. Otherwise we always init a rng to just get one random number from it... Probably not called often though and the actual (pseudo) randomness properties are not relevant.